### PR TITLE
[Doc] add stream load faq

### PR DESCRIPTION
**Pull request by amber-create**
Signed-off-by: amber-create <yangyanping@starrocks.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Review by hellolilyliuyi — DISMISSED** [review]

File: docs/faq/loading/Stream_load_faq.md [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
+- Use commands such as `sed -i '1d' filename` to delete the first row of the source data file.
+- In the load command or statement, use `-H "where: <column_name> != '<column_name>'"` to filter out the first row of the source data file. `<column_name>` is any of the column names held in the first row. Note that StarRocks first transforms and then filters the source data. Therefore, if the column names in the first row fail to be transformed into their matching destination data types, `NULL` values are returned for them. This means the destination StarRocks table cannot contain columns that are set to `NOT NULL`.
+- In the load command or statement, add `-H "max_filter_ratio:0.01"` to set a maximum error tolerance that is 1% or lower but can tolerate more than 1 error row, thereby allowing StarRocks to ignore the data transformation failures in the first row. In this case, the Stream Load job can still succeed even if `ErrorURL` is returned to indicate error rows. Do not set `max_filter_ratio` to a large value. If you set `max_filter_ratio` to a large value, some important data quality issues may not be missed.
```
Comment (evelynzhaojie) [inline docs/faq/loading/Stream_load_faq.md:10]: may not be missed. 是否有正反错误

File: docs/faq/loading/Stream_load_faq.md [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+In the preceding example, `DATE_1` can be considered to be a temporarily named column mapping the destination `DATE` column, and the final results loaded into the destination `DATE` column are computed by the `left()` function. Note that you must first list the temporary names of the source columns and then use functions to transform data. The functions supported are scalar functions, including non-aggregate functions and window functions.
+
+## 3. What do I do if my Stream Load job reports the "body exceed max size: 10737418240, limit: 10737418240" error?
+
+The size of the source data file exceeds 10 GB, which is the maximum file size supported by Stream Load. Take one of the following actions:
+
+- Use `seq -w 0 n` to split the source data file into smaller files.
+- Use `curl -XPOST http:///be_host:http_port/api/update_config?streaming_load_max_mb=<file_size>` to adjust the value of the [BE configuration item](../../administration/Configuration.md#configure-be-dynamic-parameters) to increase the maximum file size.
```
Comment (evelynzhaojie) [inline docs/faq/loading/Stream_load_faq.md:29]: 建议adjust那句也体现一下参数，不然用户需要去 url 里识别，to adjust the value of `streaming_load_max_mb`. For more information about this parameter, see xxx.

**Review by evelynzhaojie — APPROVED** [review]

**Comment by amber-create** [comment]
run starrocks_admit_test

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by amber-create** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.4

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.3

**Pull request merged**